README.md

```diff
@@ -0,0 +1,54 @@
+<h1 align="center">Welcome to bedrock-payment-http 👋</h1>
```

Comment:
Can we clean up this README to match our other projects?

Reply:
it is now a standard readme: https://github.com/digitalbazaar/bedrock-payment-http/pull/1/commits/092fc2c5a7a8331f0151df314ccbe528d4329cd3